infra/build/functions/request_build_test.py

```diff
@@ -71,9 +75,8 @@ def test_get_build_steps_no_project(self):
 
   @classmethod
   def tearDownClass(cls):
-    # TODO: replace this with a cleaner way of killing the process
```

Comment:
was this TODO addressed or why are we removing it?

Reply:
This was common code so it has been moved to test_utils.py the comment is there in that file